addons/website_event/static/src/js/systray_items/edit.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+
+import { EditSystray } from '@website/systray_items/edit';
+import { patch } from 'web.utils';
+
+patch(EditSystray.prototype, 'website_event_edit', {
+    /**
+     * @override
+     */
+    getEditActionName() {
+        if (this.websiteService.currentWebsite.metadata.mainObject.model === 'event.event') {
+            return this.env._t('Edit event');
```

Comment:
Why are we doing this in JS?
This should be done in a generic way in python/views, `ir.model.name` is exactly what we want/need: the model's name which is translatable.
And it will work everywhere, even custos.

Reply:
Indeed it would be an elegant way to do it but I think that doing it will cause a translation problem. I agree that it will work well for most languages but not for all... For example, in Japanese:
`Edit` => `編集`
`Product` => `製品情報`
but `Edit product` should be translated as `製品を編集する` which is different from `編集` + ' ' +  `製品情報`.
I don't know if my example is correct and I don't know if this problem is important enough to justify not doing what you propose above... just to let you know

Reply:
Yep, I had the same concern with this suggestion of using `ir.model.name` since the beginning. Maybe instead of "Edit XXX", we could show "XXX" next to an edit-link icon like fa-pencil and that would be clear enough :shrug: I don't know, could use @pierre-pde opinion here.

Reply:
I don't understand the japanese issue, the translatable string will be "Edit %s", which is no different from every single translatable string in Odoo, so.. Nothing too critical IMO.

But in any case I wouldn't do a "per module" override, I would make it generic, QSM suggestion is fine to me too :+1: 

Personal preference:
1. Do it generic, send "Edit %s" in translation, where %s is the main_object
2. Do it generic, "<fa-pencil> <model_name>"
3. The current way, per module override, but we won't do it anyway, generic is always better :+1:

Reply:
> the translatable string will be "Edit %s", which is no different from every single translatable string in Odoo

Yep, we can definitely fallback to "Edit %s" to be generic and handle custos but I think that a perfect translation for the main buttons of the UI is important IMO (so for "Edit product", I would definitely prefer having it translated this way rather than "Edit %s" if we have to have the "Edit" word). 

If we want to ship this before PDE is back, let's go for the solution I suggested: an icon + the model name, we can switch to (1) or (3) later if needed.

Reply:
I prefer solution 1 proposed by RDE. it's fine if a translator has to translate the word 'edit' multiple times :D 
The tool for translation helps them anyway when the recognize something.

Reply:
> I prefer solution 1 proposed by RDE. it's fine if a translator has to translate the word 'edit' multiple times :D The tool for translation helps them anyway when the recognize something.

@Guillaume-gdi FYI

Reply:
> I prefer solution 1 proposed by RDE. it's fine if a translator has to translate the word 'edit' multiple times :D

@pierre-pde You are confusing :grimacing: Solution 1 of RDE would not make the translator translate "edit" multiple times :shrug: Only the third solution kinda would but not really (would be translating sentences containing the "edit" word but would not necessarily translated to the same thing).

@Guillaume-gdi But I confirm that, for now, we go with solution 1. We may decide to override that default behavior per app later (but likely not, only I seems to think it's important so not worth it).